### ISSUE TITLE
ListBox A11Y not working

### ISSUE THREAD

**Issue by dmvvilela** [rgossiaux/svelte-headlessui#137]
I made a ListBox Inside a Dialog and A11Y features does not work. 

Tab key and keyboard up and down does not change ListboxOption inside ListboxOptions.

**Comment by NoelOConnell** [comment]
Can you provide a [repl ](https://svelte.dev/repl) example of the issue?

It works as expected for me : https://svelte.dev/repl/34783d7a5b204f8b8e78a6c5945bdee7?version=3.55.1

Up/Down key will highlight a new `ListboxOption`. The value of the `Listbox` will only change via the keyboard if the `enter` or `space` key is pressed.

![image](https://user-images.githubusercontent.com/966495/218137015-8adc3d68-6260-489a-a5fc-cb7501a876c7.png)

**Comment by dmvvilela** [comment]
Hmm.. mine is not working. Check it out here:

https://svelte.dev/repl/9c5e243eba7d42ad9ccab74f1d1f9f1e?version=3.55.1

**Comment by NoelOConnell** [comment]
I see your trying to use the `Listbox` without a `ListboxButton`. That is required for the focus management and keyboard interaction.
![image](https://user-images.githubusercontent.com/966495/218436590-6972aec8-81fc-4a32-9804-31db1440d45f.png)

From your example it looks like you want a list of items that is filterable. This would be the job of a `Combobox`.
That component is not part of the library yet. There is an open PR #126 but it is not merged yet.

I have a package on npm of which includes that PR.
https://www.npmjs.com/package/svelte-headlessui-combobox-temporary

You can find the docs for that component here
https://svelte-headlessui-git-fork-noeloconnell-combob-d68663-rgossiaux.vercel.app/docs/combobox

**Comment by dmvvilela** [comment]
Oh man, thank you so much! @NoelOConnell 🙏🏼 
Will try it now!

**Comment by dmvvilela** [comment]
Hmm Still not working: https://svelte.dev/repl/773562791ade4f439382e95976c20b7c?version=3.55.1
For some reason it doesn't even compile on REPL. But on my code tab won't work at all (neither arrow keys).
